backend/app/services/seed_qa.py

```
@@ -364,10 +410,58 @@ def build_archetypes(now: datetime | None = None) -> list[Archetype]:
+            # stale flag (the NOK/BB stale-quote case from the design evidence).
+            close_price=13.0,
+            net_change=0.5,
+            net_pct=3.85,
+            quote_time=stale_quote_time,
             marks=[
                 StubMark(15.0, far, "call", mid=0.40, delta=0.30),
             ],
         ),
+        # 9 — Dual-basis raw-loser (#422 R6 / ADR #416 Option B). A premium-
+        # softened equity holding underwater on RAW broker basis but not on the
+        # adjusted basis: raw −18.0% (past the −15% review threshold) vs adjusted
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)